api/pkg/apis/v1alpha1/managers/stage/stage-manager_test.go

```diff
@@ -46,7 +46,8 @@ func TestCampaignWithSingleMockStageLoop(t *testing.T) {
 		Stage:                "test",
 		ActivationGeneration: "1",
 		Inputs: map[string]interface{}{
-			"foo": 0,
+			"foo": "${{$output(test,foo)}}",
```

Comment:
I'm not sure what's expected behavior when activation input is the output of a stage? When the campaign is first activated, how is this value evaluated?

Reply:
Hi, Haishi. For each stage, we can have default inputs defined in the campaign stage and "runtime inputs" from activation data.
If there is no shared key in default inputs and runtime inputs, then the actual inputs for the stage provider would be a combination of the key-value pairs of the two maps.
If there are shared keys in two maps, the expected behavior should be "runtime inputs" overwrite default inputs. However, the overwrite happens in the reverse way currently.

Reply:
@lirenjie95 , @RemindD